client/src/components/ProjectSamplesTable.js

```diff
@@ -171,8 +171,13 @@ export const ProjectSamplesTable = ({
     { Header: 'Sequencing Units', accessor: 'seq_units' },
     { Header: 'Technology', accessor: 'technologies' },
     {
-      Header: 'Other Modalities',
-      accessor: ({ modalities }) => modalities.join(', ') || 'N/A'
+      Header: 'Modalities',
+      accessor: ({ modalities }) => {
+        const prefix = 'Single-cell'
+        return `${prefix} ${modalities.length > 0 ? ',' : ''} ${modalities.join(
+          `, `
+        )}`
```

Comment:
```suggestion
        return ['Single-cell', ...modalities].join(', ')}`
```

I think we can simplify this a bit since that value will always be an array. Also, we want to be sure that the comma has a trailing space.

Reply:
Suggestion is malformed, but the idea is there, I think this could be a one line method

Reply:
Thank you for this great suggestion - I've applied it at 3fa9756577c28f217faec957bc01f2a648e74e2f!